### PR TITLE
Release 2020.12.0

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updating developer documentation and release notes for 2020.12.0 release.

- **Is this PR related to a Monolith release?** Yes! 2020.12.0

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/require-authentication/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Require authentication
...
 
-It's important to note that protecting routes in your single-page app does **not** truly prevent the user from accessing those parts of your application. After all, it's JavaScript running in the browser and anyone could open the browser's developer tools and change things! Protecting routes provides a consistent and good experience for your users. The real security enforcement must be done in the API that your single-page app calls (see <GuideLink link="../use-the-access-token">Use the access token</GuideLink>). See also the [Protect your API endpoints guide](/docs/guides/protect-your-api/).
+It's important to note that protecting routes in your SPA does **not** truly prevent the user from accessing those parts of your application. After all, it's JavaScript running in the browser and anyone could open the browser's developer tools and change things! Protecting routes provides a consistent and good experience for your users. The real security enforcement must be done in the API that your single-page app calls (see <GuideLink link="../use-the-access-token">Use the access token</GuideLink>). See also the [Protect your API endpoints guide](/docs/guides/protect-your-api/).
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/require-authentication/index.md:6]:
```suggestion
It's important to note that protecting routes in your SPA does not truly prevent the user from accessing those parts of your application. After all, it's JavaScript running in the browser and anyone could open the browser's developer tools and change things! Protecting routes provides a consistent and good experience for your users. The real security enforcement must be done in the API that your single-page app calls (see <GuideLink link="../use-the-access-token">Use the access token</GuideLink>). See also the [Protect your API endpoints guide](/docs/guides/protect-your-api/).
```

**Review by jakubvul-okta — CHANGES_REQUESTED** [review]
RL System log doc should not be a part of this

File: packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.12.0
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Inclusive language and terminology](#inclusive-language-and-terminology) | December 10, 2020         |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md:9]:
```suggestion
| [Inclusive language and terminology](#inclusive-language-and-terminology) | December 9, 2020         |
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+| [New OAuth Administrator Roles API scopes](#new-oauth-administrator-roles-api-scopes) | December 10, 2020         |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md:10]:
```suggestion
| [New OAuth Administrator Roles API scopes](#new-oauth-administrator-roles-api-scopes) | December 9, 2020         |
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+| [New endpoint added to DynamicScale Rate Limits](#new-endpoint-added-to-dynamicscale-rate-limits) | December 10, 2020         |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md:11]:
```suggestion
| [New endpoint added to DynamicScale Rate Limits](#new-endpoint-added-to-dynamicscale-rate-limits) | December 9, 2020         |
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+| [Account linking for SAML IdPs is now GA in Production](#account-linking-for-saml-idps-is-now-ga-in-production) | December 10, 2020  |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md:12]:
```suggestion
| [Account linking for SAML IdPs is now GA in Production](#account-linking-for-saml-idps-is-now-ga-in-production) | December 9, 2020  |
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+| [One Time Use Refresh Token is now in Early Access (EA)](#one-time-use-refresh-token-is-now-in-early-access-ea)      | December 10, 2020         |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md:13]:
```suggestion
| [One Time Use Refresh Token is now in Early Access (EA)](#one-time-use-refresh-token-is-now-in-early-access-ea)      | December 9, 2020         |
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+| [Enhancements to Apps API for Idp Initiated Logins](#enhancements-to-apps-api-for-idp-initiated-logins) | December 10, 2020         |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md:14]:
```suggestion
| [Enhancements to Apps API for Idp Initiated Logins](#enhancements-to-apps-api-for-idp-initiated-logins) | December 9, 2020         |
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+| [Enhancements to Apps API for SAML Apps](#enhancements-to-apps-api-for-saml-apps) | December 10, 2020         |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md:15]:
```suggestion
| [Enhancements to Apps API for SAML Apps](#enhancements-to-apps-api-for-saml-apps) | December 9, 2020         |
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+| [Group object `source` property is now GA in Production](#group-object-source-property-is-now-ga-in-production) | December 10, 2020        |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md:16]:
```suggestion
| [Group object `source` property is now GA in Production](#group-object-source-property-is-now-ga-in-production) | December 9, 2020        |
```

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Comment by susanharper-okta** [comment]
> RL System log doc should not be a part of this

Done

**Review by jakubvul-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+### Enhancements to Apps API for Idp Initiated Logins
+
+The [Apps API](/docs/reference/api/apps/) can now configure the Idp Initiated Login behavior, which is also available in the Admin Console. **Note:** The Idp Initiated Login is limited to OpenID Connect clients. <!--OKTA-342821-->
+
+### Enhancements to Apps API for SAML Apps
+
+The [Apps API](/docs/reference/api/apps/) can now configure the SLO URL behavior for SAML apps, which is also available in the Admin Console.<!--OKTA-342882-->
+
+### Group object `source` property is now GA in Production
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md:54]:
```suggestion
### Group object source property is now GA in Production
```

**Pull request merged**